client_packages/LSOnline/Browsers/src/pages/characterSelect/main.js

```diff
@@ -4,8 +4,12 @@ import Vue from "vue";
 import App from "./App.vue";
 import {library} from "@fortawesome/fontawesome-svg-core";
 import {faUser} from "@fortawesome/free-solid-svg-icons/faUser";
+import {faUsers} from "@fortawesome/free-solid-svg-icons/faUsers";
+import {faChevronRight} from "@fortawesome/free-solid-svg-icons/faChevronRight";
 
 library.add(faUser);
+library.add(faUsers);
+library.add(faChevronRight);
```

Comment:
It's possible to import all these in one group. 
`import { faUser, faUsers, faChevronRight } from "@fortawesome/free-solid-svg-icons"`

and `library.add()` can take array as argument.

Sorry, I believe I could make some confusions by my code at which you based. But it's working anyway 👍